src/CodeStyle/Core/Analyzers/CodeStyleResources.resx

```diff
@@ -117,8 +117,13 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-  <data name="EmptyResource" xml:space="preserve">
-    <value>Remove this value when another is added.</value>
-    <comment>https://github.com/Microsoft/msbuild/issues/1661</comment>
+  <data name="Formatting_analyzer_message" xml:space="preserve">
```

Comment:
>Formatting_analyzer_message [](start = 14, length = 27)

Why two separate resource strings with same contents? Should we just have a single resource string "Fix_formatting" instead?

Reply:
👍

Reply:
Fixed in 031cd695031feab5d388bc8a15e297fb19a584d0